events/hfe_events_discovery.txt

```diff
@@ -19,32 +23,49 @@ country_event = {
 			auto_delete = no
 			day_zero_contact = no
 			name_list = root
-			ship_prefix = ""
+			ship_prefix = root
```

Comment:
No prefix, its intended

Reply:
I don't know why it says "ROOT". I named the Ship Prefix "Discovery".

Reply:
Then it will be Discovery Discovery

Reply:
Okay, done